frontend/src/app/shared/components/file-links/file-link-list/file-link-list-item.component.ts

```diff
@@ -106,4 +111,24 @@ export class FileLinkListItemComponent implements OnInit, AfterViewInit {
       );
     }
   }
+
+  private derivePermissionState():void {
+    switch (this.fileLink._links.permission.href) {
+      case 'urn:openproject-org:api:v3:file-links:permission:View':
```

Comment:
Is this string the same for every instance? So even for self-hosted, custom-domain instances it will start with `urn:openproject-org`?

Reply:
those constant URNs are static for the whole OP product. What I am currently missing, is some kind of central lookup in the frontend. I feel like it is very cumbersome, if we just put those into our components or templates.